init.lua

```diff
@@ -7,7 +7,10 @@ local http = minetest.request_http_api()
 
 beerchat = {
 	-- The main channel is the one you send messages to when no channel is specified
-	main_channel_name = "main",
+	main_channel_name = minetest.settings:get("beerchat.main_channel_name"),
+
+	-- Jail channel is where you put annoying missbehaving users with /force2channel
+	jail_channel_name = minetest.settings:get("beerchat.jail_channel_name"),
```

Comment:
I may be over cautious, but this is data from user config, don't want to take the chance of having empty string pass as valid.

Reply:
I think this is overcautious because when you look at how mods are misused for purpose I think it would be better to document that instead of prevent it.

As for code quality assurance and understandability I think it is bad to change settings outside of initialization method when settings are read from configuration file and are not runtime settings initialized later through some command.

If empty string can actually crash beerchat then of course check for empty string, but even then I would suggest moving that check into initialization method OR moving everything "jail" related into jail.lua file.

Reply:
I'm ok with your suggestion if other methods and commands would support it.
Let's resolve that in further step (I know you have started, just cleaning up for history)